app/uk/gov/hmrc/tariffclassificationfrontend/service/CasesService.scala

```diff
@@ -136,4 +137,29 @@ class CasesService @Inject()(appConfig: AppConfig, auditService: AuditService,
     }
   }
 
+  def addKeyword(c: Case, keyword: String)(implicit hc: HeaderCarrier): Future[Case] = {
+    c.keywords match {
+      case keywords if keywords.contains(keyword.toUpperCase) => Future.successful(c)
+      case _ =>
+        val caseToUpdate = c.copy(keywords = c.keywords + keyword.toUpperCase)
+        connector.updateCase(caseToUpdate)
+    }
+  }
+
+  def removeKeyword(c: Case, keyword: String)(implicit hc: HeaderCarrier): Future[Case] = {
+    c.keywords match {
+      case keywords if keywords.contains(keyword.toUpperCase) =>
+        val caseToUpdate = c.copy(keywords = c.keywords - keyword.toUpperCase)
+        connector.updateCase(caseToUpdate)
+      case _ => Future.successful(c)
+    }
+  }
+
+  def autoCompleteKeywords(): Future[Seq[String]] = {
```

Comment:
I suggest to remove `()` here and everywhere you call it.
If a method has no input parameters, the Scala convention is to use parenthesis (`()`) only if you return `Unit`.
See here: https://stackoverflow.com/questions/7600910/difference-between-function-with-parentheses-and-without

Reply:
Done